### DISCUSSION TITLE
Why to use Glide instead of Picasso?

### DISCUSSION THREAD

**Discussion by Vikasffic** [vikas-kmr1/Lost_n_Found#6] [Q&A] [answered]
@vickyrules  Why you use Glide while Picasso is working Fine?

**Comment by vikas-kmr1** [comment, marked as answer]
@Vikasffic 
@Vikasffic 
Glide's loading times are faster and it uses a small amount of memory 
for cache, but the library size is quite large. It, too, is easy to 
implement. Glide might be a better alternative to Picasso when memory
footprint is less of a concern or more and larger images need to be processed


Memory consumption: Glide occupies less memory as compared to Picasso. The reason being is Picasso loads full images doesn't matter what ImageView dimensions are mentioned, whereas Glide loads images from server for the ImageView size (height/width)
Library size: Glide is heavier in terms of library size as compared to Picasso.
The number of methods: Glide is having more number of methods as compared to Picasso.
GIF Support: Picasso doesn't support GIF image loading.
